src/System.Runtime.WindowsRuntime.UI.Xaml/src/System/Windows/Media/Animation/KeyTime.cs

```diff
@@ -61,7 +61,7 @@ public bool Equals(KeyTime value)
 
         public override bool Equals(object value)
         {
-            if (value == null || !(value is KeyTime))
+            if (!(value is KeyTime))
```

Comment:
I'd prefer that myself, but I adopted a policy of minimal change since I was hitting so many different places with just those minimal changes. If this clears CI I think I'll simplify all of the struct `Equals` overrides I've change in this manner along those lines, before squashing these commits.